### ISSUE TITLE
Option to hide Onboarding Message

### ISSUE THREAD

**Issue by Curly-Mo** [cynber/lemmy-instance-assistant#18]
Could you add an option in the settings to toggle off the onboarding messaging?

For new users, it's really great that the sidebar displays:
```
Your home instance is {{X}}.

▲ How to change home instance ▲
...
```
to help them figure out how to set up the extension.  

But once you've set it up, that text just becomes visual noise. It's a distraction and takes up prime real estate away from where the info about the instance/community should be.  

Thanks!  
Loving the extension so far btw. Especially the "improved 'Community not found' page" feature!

**Comment by cynber** [comment]
That sounds great, I'll see about adding this when I deal with the other wordiness issue. Thank you for the suggestion!

**Comment by cynber** [comment]
implemented, will be in v1.2.5
